pkg/operator/ceph/file/filesystem.go

```diff
@@ -108,9 +108,11 @@ func deleteFilesystem(context *clusterd.Context, cephVersion cephver.CephVersion
 		return fmt.Errorf("failed to down filesystem %s: %+v", fs.Name, err)
 	}
 
+	logger.Infof("GO TO DELETE FILESYSTEM")
```

Comment:
was this temporary for debugging? i'd suggest to remove it

Reply:
Sorry... my fault ...thank you!